.github/bin/spellcheck.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# The markdown mode of aspell does not make code be ignored. This sed command
+# removes both inline code and code blocks.
+sed -e 's/\s*`\b.*\b`//g' -e '/```.*/,/```/d' README.md -i
+aspell --home-dir=.github --personal=data/wordlist.txt --lang=en_US --mode=markdown list < README.md | sort -u > badwords
+if [ -s badwords ]
+then
+    grep -w -C 3 --color -f badwords README.md
+    echo -e "\n-----------------------------------------------------------------------\n"
```

Comment:
Using echo -e is "problematic" (does not work reliably across shells), and I'd recommend replacing it with printf. This is a nitpick level comment, and would only matter if you change shebang to #!/bin/sh. (Flags to echo are generally something I avoid, and replace it by printf.)